execution/execution_test.go

```diff
@@ -236,14 +236,14 @@ func TestExecute(t *testing.T) {
 	t.Run("Invalid transaction, Should return error", func(t *testing.T) {
 		randAddr := ts.RandAccAddress()
 		trx := tx.NewTransferTx(lockTime, randAddr, ts.RandAccAddress(),
-			ts.RandAmount(), ts.RandFee(), "invalid-tx")
+			ts.RandAmount(), ts.RandFee(), tx.WithMemo("invalid-tx"))
```

Comment:
```suggestion
			ts.RandAmount(), ts.RandFee())
```


Same here. The test declares that this is an `invalid-tx`.

I don't mention other similar cases.